README.md

```diff
@@ -2,4 +2,43 @@
 
 [![Go](https://github.com/evenlab/go-kit/actions/workflows/go.yml/badge.svg)](https://github.com/evenlab/go-kit/actions/workflows/go.yml)
 
-There are helpful lightweight and tested functionality for reuse in other projects
+### Golang KIT of helpful lightweight and tested functionality for reuse in other projects
+
+# `Makefile` Usage on Windows
+
+## `choco` Setup Chocolatey CLI
+
+### Requirements:
+* Windows 7+ / Windows Server 2003+
+* PowerShell v2+ (Not PowerShell Core yet though)(minimum is v3 for install from this website due to TLS 1.2 requirement)
+* .NET Framework 4+ (the installation will attempt to install .NET 4.0 if you do not have it installed)(minimum is 4.5 for install from this website due to TLS 1.2 requirement)
+> That's it! All you need is choco.exe (that you get from the installation scripts) and you are good to go! No Visual Studio required.
+
+### Installing Chocolatey
```

Comment:
Maybe we should be redirecting to the Choco PM installation page? If the installation method changes, we will have to edit the README (which is not good (write README according to SOLID 😅))

Reply:
I believe there is not much to change there.